### PR TITLE
Apply contribution guidelines to "AlwaysValid" rule

### PR DESCRIPTION
**Pull request by williamespindola**

### CODE REVIEW

File: tests/unit/Rules/AlwaysValidTest.php [hunk 1]
```diff
@@ -17,11 +17,20 @@
...
+ *
  * @covers \Respect\Validation\Rules\AlwaysValid
+ *
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Paulo Eduardo <pauloelr@gmail.com>
+ * @author William Espindola <oi@williamespindola.com.br>
  */
-class AlwaysValidTest extends TestCase
+final class AlwaysValidTest extends TestCase
```
Comment (batusa) [inline tests/unit/Rules/AlwaysValidTest.php]: Could you inherit `RuleTestCase` to keep consistent with the other tests?

Comment (williamespindola) [reply tests/unit/Rules/AlwaysValidTest.php]: @batusa this rule always return valid and is not possible create a teste for invalid providers. To implement RuleTestCase we need to [implement valid and invalid providers](https://github.com/Respect/Validation/blob/master/tests/library/RuleTestCase.php#L41-L53), and in this case is not possible.

File: tests/integration/rules/always-valid.phpt [hunk 2]
```diff
@@ -1,5 +1,3 @@
---TEST--
-alwaysValid()
 --FILE--
 <?php
 require 'vendor/autoload.php';
```
Comment (henriquemoody) [inline tests/integration/rules/always-valid.phpt]: This test does not cover anything with exceptions.

Could replace its content with `v::not(v::alwaysValid())->check(...)` and `v::not(v::alwaysValid())->assert(...)`?

**Comment by henriquemoody** [comment]
Merged by #985.

Thanks for this pull request, dude! 🐼

**Pull request closed without merging**